webapp/src/index.tsx

```diff
@@ -243,6 +244,10 @@ export default class Plugin {
         registry.registerWebSocketEventHandler(`custom_${pluginId}_caption`, (ev) => {
             handleCaption(store, ev);
         });
+
+        registry.registerWebSocketEventHandler(`custom_${pluginId}_host_mute`, (ev) => {
+            handleHostMute(store, ev);
+        });
```

Comment:
Do we need to update the standalone side as well?

Reply:
I don't think so -- the bot won't ever receive the message, right?

Reply:
Global widget would though

Reply:
😂 Can't believe I missed that. Thanks.